app/models/group.rb

```diff
@@ -0,0 +1,24 @@
+# == Schema Information
+#
+# Table name: groups
+#
+#  id            :bigint           not null, primary key
+#  name          :string           not null
+#  description   :text
+#  notes         :text
```

Comment:
Are there any validations that would be nice to have for a Group? I suspect `name` should be required, but I'm not sure about `description` or `notes`.

Reply:
notes/description can be blank, these are just for the user benefit.  I've added validates :name, presence: true to lock that down (beyond just having the not null db constraint)